tests/forms_tests/tests/test_fields.py

```diff
@@ -722,6 +722,10 @@ class DurationForm(Form):
             str(f['duration'])
         )
 
+    def test_durationfield_3(self):
```

Comment:
I don't think it's much of a "feature" (if you meant for 1.10) and I didn't see any justification for a backport (if you meant 1.9.2).